### ISSUE TITLE
Example is not working..

### ISSUE THREAD

**Issue by parkjaeming** [liulab-dfci/TRUST4#64]
Hi, I'm trying to run my paired fastq files to find the TCR/BCR sequences with TRUST4. So, I followed the example, but I got the error below.

./run-trust4 -b example/example.bam -f hg38_bcrtcr.fa --ref human_IMGT+C.fa 
[Mon Jul 26 15:18:18 2021] TRUST4 begins.
[Mon Jul 26 15:18:18 2021] SYSTEM CALL: /data1/jaemin_1/TRUST4/bam-extractor -b example/example.bam -t 1 -f hg38_bcrtcr.fa -o TRUST_example_toassemble 
Can't exec "/data1/jaemin_1/TRUST4/bam-extractor": No such file or directory at ./run-trust4 line 48.
system /data1/jaemin_1/TRUST4/bam-extractor -b example/example.bam -t 1 -f hg38_bcrtcr.fa -o TRUST_example_toassemble  failed: -1 at ./run-trust4 line 48.

I got the same error in the fastq example and my own data. How can I solve this problem?

**Comment by mourisl** [comment]
run-trust4 is a wrapper, have you run "make" to compile and generate the TRUST4 executables?

**Comment by parkjaeming** [comment]
I didn't..! Actually, I don't fully understand how I can make TRUST4 executable... Can I get some advice?

**Comment by parkjaeming** [comment]
I didn't..!
I just ran the 'make' command, and I got annotator, bam-extractor, fastq-extractor, trust4.

$ ls
AlignAlgo.hpp         BuildDatabaseFa.pl       FilterAnnotatedAssembly.pl  LICENSE.txt      scripts
alignments.hpp        BuildImgtAnnot.pl        hg19_bcrtcr.fa              main.cpp         SeqSet.hpp
annotator             BuildImgtVquestAnnot.pl  hg38_bcrtcr.fa              main.o           SimpleVector.hpp
Annotator.cpp         Cgene.list               human_IMGT+C.fa             Makefile         trust4
Annotator.o           defs.h                   human_vdjc.list             mouse            trust-barcoderep.pl
bam-extractor         example                  KmerCode.hpp                ReadFiles.hpp    trust-simplerep.pl
BamExtractor.cpp      fastq-extractor          KmerCount.hpp               README.md        trust-smartseq.pl
BamExtractor.o        FastqExtractor.cpp       KmerIndex.hpp               run-trust4
BarcodeCorrector.hpp  FastqExtractor.o         kseq.h                      samtools-0.1.19

So, I tried the example again, but I got a similar error..

./run-trust4 -b example/example.bam -f hg38_bcrtcr.fa --ref human_IMGT+C.fa 
[Mon Jul 26 16:59:03 2021] TRUST4 begins.
[Mon Jul 26 16:59:03 2021] SYSTEM CALL: /data1/jaemin_1/TRUST4/bam-extractor -b example/example.bam -t 1 -f hg38_bcrtcr.fa -o TRUST_example_toassemble 
[Mon Jul 26 16:59:04 2021] Start to extract candidate reads from bam file.
system /data1/jaemin_1/TRUST4/bam-extractor -b example/example.bam -t 1 -f hg38_bcrtcr.fa -o TRUST_example_toassemble  failed: 139 at ./run-trust4 line 48.
